scripts/sysman/memoryBwCounterValidBits.yml

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Bandwidth Counter Valid Bits"
```

Comment:
Can we use "Memory "Bandwidth here, just to be specific ?

Reply:
done